torchvision/prototype/datasets/__init__.py

```diff
@@ -7,10 +7,9 @@
         "Note that you cannot install it with `pip install torchdata`, since this is another package."
     ) from error
 
-
 from . import decoder, utils
+from ._home import home
 
 # Load this last, since some parts depend on the above being loaded first
-from ._api import register, _list as list, info, load
+from ._api import register, _list as list, info, load  # usort: skip
```

Comment:
Just curious, what's the reason we need to load these after _home?

Reply:
To avoid cyclic imports. Importing anything from `_api` here will try to import `home`:

https://github.com/pytorch/vision/blob/0c0a6a448a001b91e0d12f50225b68372bed2354/torchvision/prototype/datasets/_api.py#L6